### ISSUE TITLE
paddlegpu显示安装成功，实际是CPU版本

### ISSUE THREAD

**Issue by hnlslyp** [PaddlePaddle/Paddle#46571]
### bug描述 Describe the Bug

centos 6
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.33.01    Driver Version: 440.33.01    CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
#################
  显卡正常
#################

 /opt/compiler/gcc-8.2/lib64/ld-linux-x86-64.so.2 --library-path /opt/compiler/gcc-8.2/lib64/:/usr/local/openssl/lib/:/ssd4/liuyaping/python38/lib:/home/work/cuda-10.2/lib64/:/home/work/cuda-10.2/lib64:/lib64/:/usr/lib64/ ./python38/bin/python3.8 -m pip install ./env/paddlepaddle_gpu-2.3.2.post116-cp38-cp38-linux_x86_64.whl --force-reinstall
Looking in indexes: http://pip.baidu.com/root/baidu/+simple/
Processing ./env/paddlepaddle_gpu-2.3.2.post116-cp38-cp38-linux_x86_64.whl
Collecting paddle-bfloat==0.1.7
  Using cached http://pip.baidu.com/root/baidu/packages/packages/76/d7/ba0e1aeec33e20c78af5cf2fdbb7e7cabfe4679557e68759a17c97e03540/paddle_bfloat-0.1.7-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (385 kB)
Collecting opt-einsum==3.3.0
  Using cached http://pip.baidu.com/root/baidu/packages/packages/bc/19/404708a7e54ad2798907210462fd950c3442ea51acc8790f3da48d2bee8b/opt_einsum-3.3.0-py3-none-any.whl (65 kB)
Collecting astor
  Using cached http://pip.baidu.com/root/baidu/packages/packages/c3/88/97eef84f48fa04fbd6750e62dcceafba6c63c81b7ac1420856c8dcc0a3f9/astor-0.8.1-py2.py3-none-any.whl (27 kB)
Collecting protobuf<=3.20.0,>=3.1.0
  Using cached http://pip.baidu.com/root/baidu/packages/packages/88/88/cd55f87e896b82a3aba8e6c0affc077de51f7321cf730622b17ef7b0f69c/protobuf-3.20.0-cp38-cp38-manylinux_2_5_x86_64.manylinux1_x86_64.whl (1.0 MB)
Collecting requests>=2.20.0
  Using cached http://pip.baidu.com/root/baidu/packages/packages/ca/91/6d9b8ccacd0412c08820f72cebaa4f0c0441b5cda699c90f618b6f8a1b42/requests-2.28.1-py3-none-any.whl (62 kB)
Collecting numpy>=1.13
  Using cached http://pip.baidu.com/root/baidu/packages/packages/d6/e2/bed33bdbf513cd6d3fcb4377792ef1b8aad941da542a191e1e2a98c6621f/numpy-1.23.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.1 MB)
Collecting six
  Using cached http://pip.baidu.com/root/baidu/packages/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting decorator
  Using cached http://pip.baidu.com/root/baidu/packages/packages/d5/50/83c593b07763e1161326b3b8c6686f0f4b0f24d5526546bee538c89837d6/decorator-5.1.1-py3-none-any.whl (9.1 kB)
Collecting Pillow
  Using cached http://pip.baidu.com/root/baidu/packages/packages/20/cb/261342854f01ff18281e97ec8e6a7ce3beaf8e1091d1cebd52776049358d/Pillow-9.2.0-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)
Collecting certifi>=2017.4.17
  Using cached http://pip.baidu.com/root/baidu/packages/packages/1d/38/fa96a426e0c0e68aabc68e896584b83ad1eec779265a028e156ce509630e/certifi-2022.9.24-py3-none-any.whl (161 kB)
Collecting charset-normalizer<3,>=2
  Using cached http://pip.baidu.com/root/baidu/packages/packages/db/51/a507c856293ab05cdc1db77ff4bc1268ddd39f29e7dc4919aa497f0adbec/charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached http://pip.baidu.com/root/baidu/packages/packages/6f/de/5be2e3eed8426f871b170663333a0f627fc2924cc386cd41be065e7ea870/urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
Collecting idna<4,>=2.5
  Using cached http://pip.baidu.com/root/baidu/packages/packages/fc/34/3030de6f1370931b9dbb4dad48f6ab1015ab1d32447850b9fc94e60097be/idna-3.4-py3-none-any.whl (61 kB)
Installing collected packages: paddle-bfloat, urllib3, six, protobuf, Pillow, numpy, idna, decorator, charset-normalizer, certifi, astor, requests, opt-einsum, paddlepaddle-gpu
  Attempting uninstall: paddle-bfloat
    Found existing installation: paddle-bfloat 0.1.7
    Uninstalling paddle-bfloat-0.1.7:
      Successfully uninstalled paddle-bfloat-0.1.7
  Attempting uninstall: urllib3
    Found existing installation: urllib3 1.26.12
    Uninstalling urllib3-1.26.12:
      Successfully uninstalled urllib3-1.26.12
  Attempting uninstall: six
    Found existing installation: six 1.16.0
    Uninstalling six-1.16.0:
      Successfully uninstalled six-1.16.0
  Attempting uninstall: protobuf
    Found existing installation: protobuf 3.20.0
    Uninstalling protobuf-3.20.0:
      Successfully uninstalled protobuf-3.20.0
  Attempting uninstall: Pillow
    Found existing installation: Pillow 9.2.0
    Uninstalling Pillow-9.2.0:
      Successfully uninstalled Pillow-9.2.0
  Attempting uninstall: numpy
    Found existing installation: numpy 1.23.3
    Uninstalling numpy-1.23.3:
      Successfully uninstalled numpy-1.23.3
  Attempting uninstall: idna
    Found existing installation: idna 3.4
    Uninstalling idna-3.4:
      Successfully uninstalled idna-3.4
  Attempting uninstall: decorator
    Found existing installation: decorator 5.1.1
    Uninstalling decorator-5.1.1:
      Successfully uninstalled decorator-5.1.1
  Attempting uninstall: charset-normalizer
    Found existing installation: charset-normalizer 2.1.1
    Uninstalling charset-normalizer-2.1.1:
      Successfully uninstalled charset-normalizer-2.1.1
  Attempting uninstall: certifi
    Found existing installation: certifi 2022.9.24
    Uninstalling certifi-2022.9.24:
      Successfully uninstalled certifi-2022.9.24
  Attempting uninstall: astor
    Found existing installation: astor 0.8.1
    Uninstalling astor-0.8.1:
      Successfully uninstalled astor-0.8.1
  Attempting uninstall: requests
    Found existing installation: requests 2.28.1
    Uninstalling requests-2.28.1:
      Successfully uninstalled requests-2.28.1
  Attempting uninstall: opt-einsum
    Found existing installation: opt-einsum 3.3.0
    Uninstalling opt-einsum-3.3.0:
      Successfully uninstalled opt-einsum-3.3.0
  Attempting uninstall: paddlepaddle-gpu
    Found existing installation: paddlepaddle-gpu 2.3.2.post116
    Uninstalling paddlepaddle-gpu-2.3.2.post116:
      Successfully uninstalled paddlepaddle-gpu-2.3.2.post116
Successfully installed Pillow-9.2.0 astor-0.8.1 certifi-2022.9.24 charset-normalizer-2.1.1 decorator-5.1.1 idna-3.4 numpy-1.23.3 opt-einsum-3.3.0 paddle-bfloat-0.1.7 paddlepaddle-gpu-2.3.2.post116 protobuf-3.20.0 requests-2.28.1 six-1.16.0 urllib3-1.26.12
##############
如上信息，显示paddle安装成功
##############

 /opt/compiler/gcc-8.2/lib64/ld-linux-x86-64.so.2 --library-path /opt/compiler/gcc-8.2/lib64/:/usr/local/openssl/lib/:/ssd4/liuyaping/python38/lib:/home/work/cuda-10.2/lib64/:/home/work/cuda-10.2/lib64:/lib64/:/usr/lib64/ ./python38/bin/python3.8 -c "import paddle;paddle.utils.run_check()"
Running verify PaddlePaddle program ...
PaddlePaddle works well on 1 CPU.
W0927 21:26:19.645772 16842 fuse_all_reduce_op_pass.cc:76] Find all_reduce operators: 2. To make the speed faster, some all_reduce ops are fused during training, after fusion, the number of all_reduce ops is 2.
PaddlePaddle works well on 2 CPUs.
PaddlePaddle is installed successfully! Let's start deep learning with PaddlePaddle now.

#################
实际检测仍是CPU版本
#################

**Comment by hnlslyp** [comment]
补充下，Torch工作正常
bash-4.1$ /opt/compiler/gcc-8.2/lib64/ld-linux-x86-64.so.2 --library-path /opt/compiler/gcc-8.2/lib64/:/usr/local/openssl/lib/:/ssd4/liuyaping/python38/lib:/home/work/cuda-10.2/lib64/:/home/work/cuda-10.2/lib64:/lib64/:/usr/lib64/ ./python38/bin/python3 -c "import torch;print(torch.__version__)"
1.12.1+cu102
bash-4.1$ /opt/compiler/gcc-8.2/lib64/ld-linux-x86-64.so.2 --library-path /opt/compiler/gcc-8.2/lib64/:/usr/local/openssl/lib/:/ssd4/liuyaping/python38/lib:/home/work/cuda-10.2/lib64/:/home/work/cuda-10.2/lib64:/lib64/:/usr/lib64/ ./python38/bin/python3 -c "import torch;print(torch.cuda.is_available())"
True

**Comment by zhaoyinglia** [comment]
你好，建议可以更新到 centos 6 再安装一下试试。

**Comment by zhaoyinglia** [comment]
你好，根据官网可以更新到 centos 7 再安装一下试试。

<img width="606" alt="image" src="https://user-images.githubusercontent.com/86812880/192718955-752de4c4-6118-448b-94ab-159e541a9d93.png">

**Comment by hnlslyp** [comment]
组内公用机器，不能随便改内核版本...

**Comment by dingjiaweiww** [comment]
你好，飞桨本机安装最低支持centos 7，你可以使用docker 镜像安装：https://www.paddlepaddle.org.cn/documentation/docs/zh/install/docker/linux-docker.html

**Comment by dingjiaweiww** [comment]
如果你安装成功的话，你运行下这个代码看看会输出什么，这个API 是用来检测你所安装的安装包能不能在GPU 上运行，paddle.device.is_compiled_with_cuda()

<img width="471" alt="image" src="https://user-images.githubusercontent.com/23093488/194748361-99702475-8cb2-4b45-9748-4b624d036742.png">

**Comment by dingjiaweiww** [comment]
如果输出是支持GPU 运行的话，可以再按照以下步骤检查一下：
1、cuda版本和安装包的cuda版本是否对应
2、cuda相关的环境变量是否指向了想要的cuda版本
